.github/workflows/deploy.yaml

```diff
@@ -61,9 +55,9 @@ jobs:
           context: .
           push: true
           build-args: |
-            OKTETO_BIN_TAG=${{ github.event.inputs.bin_tag }}
+            OKTETO_BIN_TAG=${{ github.event.release.tag_name }}
           tags: |
-            okteto/bin:${{ github.event.inputs.bin_tag }}
+            okteto/bin:${{ github.event.release.tag_name }}
           platforms: linux/amd64,linux/arm64
```

Comment:
This should do the trick based in https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#release